lldb/test/API/lit.cfg.py

```diff
@@ -250,6 +250,9 @@ def delete_module_cache(path):
 if is_configured("dsymutil"):
     dotest_cmd += ["--dsymutil", config.dsymutil]
 
+if is_configured("make"):
+    dotest_cmd += ["--make", config.make]
+
```

Comment:
`--make` is added if `LLDB_TEST_MAKE` is defined explicitly or `LLDB_ENFORCE_STRICT_TEST_REQUIREMENTS=ON`

Reply:
Changed in follow-up patch https://github.com/llvm/llvm-project/pull/111531/commits/3f6b93d932d9755c668d31b9c75c152bb3599bbe We now always detect `make`.